src/pi-gen.ts

```diff
@@ -51,7 +51,7 @@ export class PiGen {
   }
 
   async build(verbose = false): Promise<exec.ExecOutput> {
-    const dockerOpts = this.getStagesAsDockerMounts()
+    const dockerOpts = `${this.getStagesAsDockerMounts()} ${this.config.dockerOpts}`.trim()
```

Comment:
I'm not sure it's a good idea to allow user input to override the action's internal Docker options. Is that a specific requirement you have? Otherwise, the order should be switched:
```suggestion
    const dockerOpts = `${this.config.dockerOpts} ${this.getStagesAsDockerMounts()}`.trim()
```

Reply:
No, I don't have that as a requirement, I just need to mount additional paths into the container. However I guess you could argue either way, that overriding some choice the action makes could also become useful. I followed your suggestion in #16  anyways :)